clar2wasm/src/tools.rs

```diff
@@ -427,6 +427,45 @@ pub fn crosscheck_validate<V: Fn(Value)>(snippet: &str, validator: V) {
     validator(value)
 }
 
+pub fn crosscheck_multi_contract(
+    contracts: &[(ContractName, &str)],
+    expected: Result<Option<Value>, Error>,
+) {
+    // compiled version
+    let mut env = TestEnvironment::default();
+    let compiled_results: Vec<_> = contracts
+        .iter()
+        .map(|(name, snippet)| env.init_contract_with_snippet(name, snippet))
+        .collect();
+
+    // TODO: this is the interpreted version and the comparison with the compiled results.
+    //       It need a fix in `interpret_contract_with_snippet` to work.
```

Comment:
Do we have an issue to track this necessary fix? If yes lets add the issue number on the comment.

Reply:
Not yet

Reply:
Not gonna open an issue, I'll fix it when I'll finish #287